picard/ui/options/plugins.py

```diff
@@ -230,6 +234,22 @@ def plugin_updated(self, plugin_name):
                 self.change_details()
                 break
 
+    def plugin_uninstall(self):
+            plugin = self.items[self.ui.plugins.selectedItems()[0]]
+            buttonReply = QtWidgets.QMessageBox.question(self, 'Confirmation',
+                    "Do you really want to uninstall the plugin %s ?" % plugin.name,
+                    QtWidgets.QMessageBox.Yes | QtWidgets.QMessageBox.No, QtWidgets.QMessageBox.No)
+            if buttonReply == QtWidgets.QMessageBox.Yes:
+                self.tagger.pluginmanager.remove_plugin(plugin.module_name)
+                self.ui.plugins.itemWidget(self.ui.plugins.currentItem(), ACTION_COLUMN_NO).setText(_("Uninstalled"))
```

Comment:
You have to disable the button too and also uncheck the enable/disable checkbox (if it was checked).